.config/git/config

```diff
@@ -8,12 +8,13 @@
   ff = only
   rebase = true
 [core]
+  pager = delta
   editor = nvim
   excludesfile = ~/.config/git/global_ignore
+[interactive]
+  diffFilter = delta --color-only
```

Comment:
For interactive git commands involving git patches, add syntax highlighting without modifying anything else.
https://git-scm.com/docs/git-config#Documentation/git-config.txt-interactivediffFilter